fluent-bit/config.sls

```diff
@@ -3,9 +3,9 @@
 
 {% from "fluent-bit/map.jinja" import fluent_bit with context %}
 
-fluent-bit-config:
+{{ fluent_bit.pkg }}-config:
```

Comment:
It's better to use `{{ sls }}` so it uses the directory name whatever is it.

Reply:
You are right, but package name is already sanitized, and directory can have any characters which can break things. A will give you still the output what you are installing for "version" even if future fork of anything-bit will be released.